draft-opsarea-rfc5706bis.md

```diff
@@ -53,6 +53,16 @@ author:
     fullname: Ran Chen
     organization: ZTE
     email: chen.ran@zte.com.cn
+
+normative:
+
+informative:
```

Comment:
Heh.  This will conflict with my tooling text.  Note to resolve.

Reply:
You will see when you will merge :-)